Cargo.toml

```diff
@@ -8,6 +8,7 @@ keywords = ["cfb", "storage", "structured"]
 license = "MIT"
 readme = "README.md"
 edition = "2018"
+exclude = ["/tests"]
```

Comment:
I guess `cargo vendor` is a bit less convenient than it would otherwise be, and users that want to fork their dependency need to do a bit more work. My main motivation was to prevent the binary cfb files needed for tests to bloat `.cargo` and waste bandwidth. I think I'll revert this.